### ISSUE TITLE
Cannot delete a channel whos name is a float

### ISSUE THREAD

**Issue by thistehneisen** [mattermost/mmctl#563]
e.g.:

```
root@mm:/opt/mattermost# bin/mmctl channel delete Team:13.41417518828205
Unable to find channel 'Team:13.41417518828205'
```

The channel does exist, though. Also, a wildcard support would be appreciated.

**Comment by isacikgoz** [comment]
Thanks for the feedback @thistehneisen created the issue for the bug report. Here is the ticket: https://mattermost.atlassian.net/browse/MM-48056

For the wildcard, can you elaborate more on that? Do you think something like a command below:
```sh
mmctl channel delete Team:*
```

**Comment by thistehneisen** [comment]
Yes, @isacikgoz, exactly as in the case of your example, all Teams would be deleted.

**Comment by thistehneisen** [comment]
That could be very much true, just a feature suggestion my side additional to the issue itself.

**Comment by hanzei** [comment]
Closing this ticket as https://github.com/mattermost/mattermost/issues/21874 has been closed
